contracts/AssetPool.sol

```diff
@@ -66,6 +70,12 @@ contract AssetPool is Ownable {
         uint256 timestamp
     );
     event WithdrawalTimedOut(address indexed underwriter, uint256 timestamp);
+    event UpgradeToNewAssetPoolCompleted(
```

Comment:
Maybe just `AssetPoolUpgraded`? This is just a one-step process, so there is no initiation & completion.

Reply:
Done